corehq/apps/data_analytics/management/commands/export_form_impact_data.py

```diff
@@ -40,28 +43,33 @@ def handle(self, month_year, file_path, **options):
                 'multiple_form_types?'
             ])
 
-            for domain_object in Domain.get_all():
-                domain_name = domain_object.name
+            for domain in with_progress_bar(Domain.get_all(include_docs=False)):
+                domain_name = domain['key']
                 user_ids = CommCareUser.ids_by_domain(domain=domain_name)
-                buckets_dict = get_forms_for_users(domain_object.name, user_ids, start_date, end_date)
-                for user_id, bucket in buckets_dict.iteritems():
-                    has_case = bool(
-                        filter(
-                            lambda h: h.get('form', {}).get('case'),
-                            bucket.top_hits_user_submissions.hits
+                for users in chunked(user_ids, 100):
+                    forms = get_forms_for_users(domain_name, users, start_date, end_date)
```

Comment:
I think some forms can get pretty long. Can you add a parameter to specify the fields you need?